src/app/settings/containers/settings/settings.component.html

```diff
@@ -12,6 +12,10 @@
     [apiDocumentationLink]="apiDocumentationLink"
     (regenerateKeys)="onRegenerateKeys()"
   ></cs-api-settings>
+  <cs-vm-settings
+    [settings]="settings$ | async"
+    (updateKeyboardLayout)="keyboardChange($event)"
```

Comment:
Make it looks like other bindings in the components.
It looks that its logic is different